src/store/index.ts

```diff
@@ -154,3 +136,56 @@ async function fetchAttestationToken(): Promise<string> {
     }
   }
 }
+
+/**
+ * Helper: Post a sealed request, and unseal the response.
+ *
+ * @param context - Vuex action context
+ * @param url - The endpoint URL to post to
+ * @param value - The unsealed value to seal and post
+ * @param encrypt - Function to encrypt (seal) the value with
+ * @returns - The unsealed response bytes
+ */
+async function sealedPost<T>(
+  context: ActionContext<State, State>,
```

Comment:
I would prefer passing in the public key value instead of action context.

Reply:
Updated: [aa891f6..7c17610](https://github.com/registreerocks/trusted-compute-web/compare/aa891f6..7c17610)